src/ui/src/app/deployment-item/deployment-item.component.scss

```diff
@@ -0,0 +1,60 @@
+// Import your custom theme
+@import '../../theme.scss';
+
+.deployment_item {
```

Comment:
Some of the selectors of this class are pretty similar to the `chart-details`. Can we extract common parts to new components? If we create them, we won't need to duplicate the logic and UI.